Themis/Views/Assessment/CorrectionSidebar/ProblemStatementCellView.swift

```diff
@@ -20,9 +20,35 @@ struct ProblemStatementCellView: View {
             Text("Problem Statement")
                 .font(.largeTitle)
 
+            ZStack {
+                Divider()
+                    .frame(height: 5)
+                    .overlay(.gray)
+                HStack {
+                    ForEach(vm.bundledTests, id: \.testCase) { bundledTest in
+                        if bundledTest != vm.bundledTests.first {
+                            Spacer()
+                        }
+                        if bundledTest.passed {
+                            Image("TestPassedSymbol")
+                                .resizable()
+                                .scaledToFit()
+                                .frame(maxWidth: 30, maxHeight: 30)
+                                .background(colorScheme == .light ? .white : .black)
```

Comment:
Just use a dynamic color. In you case `.background(Color(.systemBackground))` for example

Reply:
Thanks!